### DISCUSSION TITLE
6660 - custom PPPoE termination of the DOCSIS connection

### DISCUSSION THREAD

**Discussion by Dukat-Gul** [Freetz-NG/freetz-ng#991] [Unerfüllte Wünsche]
I am interested to know if Freetz-NG could support a 6660 Cable router with a manual or non-standard PPPoE support?
The current AVM firmware seems to have a DOCSIS connection without any ability to input PPP username/password.
I am about to try the lab - as I see the changelog indicate some new ISP connection methods.

**Comment by NDiGH** [comment]
> I am interested to know if Freetz-NG could support a 6660 Cable router with a manual or non-standard PPPoE support?

No.
